h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -794,9 +797,15 @@ protected void ignoreBadColumns(int npredictors, boolean expensive){
    * Checks response variable attributes and adds errors if response variable is unusable.
    */
   protected void checkResponseVariable() {
+    if (_response != null && !ArrayUtils.contains(ALLOWED_RESPONSE_TYPES, _response.get_type())) {
+      StringBuilder allowedTypesStringBuilder = new StringBuilder();
```

Comment:
This can be simplified - this can be completely static; there is nothing dynamic here.

I would not add a function translateType to Vec.